### ISSUE TITLE
Add colors

### ISSUE THREAD

**Issue by mgifford** [mgifford/daily-dap#53]
This table should have colors for how close to 100 they are. 
https://mgifford.github.io/daily-dap/docs/reports/daily/2026-03-16/index.html#top-urls-heading

Looking at:

URL | Traffic | Scan status | CWV | Performance | Accessibility | Best Practices | SEO | Total findings | Critical/Serious | Failure reason | Axe details
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

I would like to see each of these columns to have a color gradient where the darkest is 100% and the lightest is 0

CWV | Performance | Accessibility | Best Practices | SEO
-- | -- | -- | -- | --

Each of these columns could be a different color.

Purple, green, blue, orange, yellow.

something like that.

It would be easier to see where are the outliers.

This is much like how Google Sheets does this.
